.github/workflows/demo_test.yml

```diff
@@ -376,7 +376,7 @@ jobs:
       run: docker exec openvino_test bash -c "cd /root/occlum; OCCLUM_RELEASE_BUILD=y make install"
 
     - name: Run openVINO benchmark
-      run: docker exec openvino_test bash -c "cd /root/demos/openvino && SGX_MODE=SIM ./run_benchmark_on_occlum.sh"
+      run: docker exec openvino_test bash -c "cd /root/demos/openvino && cp -f /root/occlum/demos/openvino/run_benchmark_on_occlum.sh . && SGX_MODE=SIM ./run_benchmark_on_occlum.sh"
```

Comment:
This line can be used to make sure the demo can pass. However, before this commit is actually merged, this line can be removed because if we changed the OpenVINO demo, a new OpenVINO demo image will/should be created and used in the successive tests.